sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -474,7 +474,7 @@ private static boolean initConfigurations(final @NotNull SentryOptions options)
       options.addPerformanceCollector(new JavaMemoryCollector());
     }
 
-    if (options.isEnableBackpressureHandling()) {
+    if (options.isEnableBackpressureHandling() && Platform.isJvm()) {
```

Comment:
Since there's currently no way of disabling backpressure for Android via Metadata I only enable it for backend. There usually shouldn't be as high a volume of transactions on Android.

Reply:
We decided in Backend Daily it's OK to disable backpressure management for Android. If we ever want it we'll have to make it configurable and probably do a separate test phase before enabling it by default there.